pkg/util/sets/set.go

```diff
@@ -233,6 +243,11 @@ func (s Set[T]) IsEmpty() bool {
 	return len(s) == 0
 }
 
+// IsNotEmpty indicates whether the set is the not empty set.
+func (s Set[T]) IsNotEmpty() bool {
```

Comment:
nit: I would prefer not to add this, it just gives users two ways to do the same thing `!x.IsEmpty()` or `x.IsNotEmpty()`. Keeping just one ensures consistency